test/unit/proof/whitespace.h

```diff
@@ -0,0 +1,34 @@
+/*********************                                                        */
+/*! \file whitespace.h
```

Comment:
Can we maybe change the filename to `utils.h` s.t. we can (potentially) add other helper functions here?

Reply:
Sounds good.